dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/cache/DefaultCacheProvider.java

```diff
@@ -123,13 +123,33 @@ private long orZeroInTestRun( long value )
         return cache;
     }
 
+    private long getSize1()
+    {
+        return 1;
+    }
+
+    private long getSize100()
+    {
+        return (long) this.cacheFactor * 100;
+    }
+
+    private long getSize1k()
+    {
+        return (long) this.cacheFactor * 1_000;
+    }
+
+    private long getSize10k()
```

Comment:
Is 10k really needed? We could be more stringent with the sizes?

Reply:
I would leave that unchanged for now.
I am not sure how these values were chosen. I am trying to change as less as possible as we are close to the freeze